src/server/transaction.h

```diff
@@ -329,16 +329,16 @@ class Transaction {
     char pad[46];  // to make sure PerShardData is 64 bytes and takes full cacheline.
 
     uint32_t arg_start = 0;  // Indices into args_ array.
-    uint16_t arg_count = 0;
-
-    // Accessed within shard thread.
-    // Bitmask of LocalState enums.
-    uint16_t local_mask = 0;
+    uint32_t arg_count = 0;
```

Comment:
I hope you grepped that uint16_t isn't used anywhere else 🙂

Reply:
eeh, now i did :)